content/Watsonx/WatsonxData/102.md

```diff
@@ -150,7 +150,7 @@ In this section you will explore the Data manager page and create your own schem
    Additionally, Presto itself (and by extension watsonx.data) includes the TPC-H data set. TPC-H is a decision support benchmark maintained by the Transaction Processing Council (TPC). It is intended to mimic a real-world business workload involving a number of ad-hoc queries running while concurrent modifications are made to the data. The dataset that supports this benchmark includes information on customers, suppliers, orders, part numbers, and more. Datasets of different scale (size) can be generated to test workloads at different scale.
 
    The TPC-H sample data (and other sample data) can be found in the System and benchmarking data section, which is below the Catalogs associated section.
-
+<QuizAlert text="Quiz material: explore all options by expanding other catalogs;"/>
```

Comment:
Sentence should be ended with a period instead of a semicolon (or neither)?

Reply:
done